# Unit 14: **Basic postulates and definitions of chemical kinetics. Kinetics of simple reactions.**

# 1. Theory

# 2. Examples

## Problem 1

The second-order reaction $ A + B -> P$ is carried out in a solution with initial concentrations $[A]_0 = 0.050 M$ and $[B]_0 = 0.080 M$. After 1 hour, the concentration of substance A decreased to 0.020 M. Calculate the rate constant and half-lives of both substances.

### Solution

We use the formula for $[A]_0 \neq [B]_0$:

$$
k\ t = \frac{1}{[A]_0 - [B]_0} ln( \frac{[A][B]_0}{[B][A]_0} )
$$

In [1]:
import math
import matplotlib.pyplot as plt
import numpy as np

A0 = 0.05 # M
A = 0.02 # M

# How much of x reacted:
x = A0 - A

B0 = 0.080 # M
B = B0 - x  # the amount of consumed B is as much as A

# 
t = 1.0 # h
k = math.log(A * B0 / (A0*B)) /(t * (A0 - B0) )

print(F"k = {k} mol/(L*h) = {k/60.0} mol/(L*min)")

# To compute half-lives, we set the values of the current concentrations of A or B to half of the initial values
# First for A
A = 0.5 * A0
x = A0 - A
B = B0 - x

# Using the same equation as before, but solve for t:
tA = math.log(A * B0 / (A0*B)) /(k * (A0 - B0) )
print(F"Half-life of A is = {tA} = {tA * 60.0} min")

# Next for B
B = 0.5 * B0
x = B0 - B
A = A0 - x

# Using the same equation as before, but solve for t:
tB = math.log(A * B0 / (A0*B)) /(k * (A0 - B0) )
print(F"Half-life of B is = {tB} h = {tB * 60.0} min")

k = 14.876236754280654 mol/(L*h) = 0.2479372792380109 mol/(L*min)
Half-life of A is = 0.7135624785995228 = 42.81374871597137 min
Half-life of B is = 2.053141859752694 h = 123.18851158516165 min


So:

k = 12.95 $\frac{L}{mol*h}$ = 0.248 $\frac{L}{mol*min}$

$\tau_{\frac{1}{2}}(A) = $0.71 h = 42.8 min

$\tau_{\frac{1}{2}}(B) = $2.05 h = 123 min

# 3. Problems

# 4. Solutions